Controller/Backend/PlentyConnector.php

```diff
@@ -269,6 +273,57 @@ public function getItemWarehousesAction()
         ]);
     }
 
+    public function syncItemAction()
+    {
+        $success = false;
+        $itemId = (int) $this->Request()->get('item_id');
+
+        if (!is_int($itemId)) {
```

Comment:
Solltest du hier nicht auf Empty prüfen da du vorher dem Wert in einen integer wandelst? Im Fehlerfall sollte auch eine growl Message im backend sichtbar sein :)